cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/sv/GMLSvDataReader.java

```diff
@@ -108,7 +109,12 @@ private void processJsonFile() {
             if (samples != null) {
                 for (GMLCnvIntragenicVariant cnv : result.getCnvIntragenicVariantsGml()) {
                     for (String sampleId : samples) {
-                        CVRSvRecord record = new CVRSvRecord(cnv, sampleId);
+                        CVRSvRecord record = svUtilities.makeCvrSvRecordFromGmlCnvIntragenicVariant(cnv, sampleId);
+                        try {
+                            svUtilities.populateEventInfoFromGmlCnvIntragenicVariant(record);
```

Comment:
can we just call this from inside the `.makeCvrSvRecordFromGmlCnvIntragenicVariant` function? seems like this will always be coupled together?